### PR TITLE
chapter 8

### PR DESCRIPTION
**Pull request by daniadore**

### CODE REVIEW

**Comment by daniadore** [comment]
Ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/sessions.scss:1]: please delete this file

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
```
Comment (anlvq-1058) [inline app/controllers/application_controller.rb:2]: you don't need this line please delete it

File: test/controllers/sessions_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/sessions_controller_test.rb:1]: You don't need do the test please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by daniadore** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**